src/SDK/Resource/ResourceInfoFactory.php

```diff
@@ -38,17 +43,21 @@ public static function defaultResource(): ResourceInfo
 
         if (in_array(Values::VALUE_ALL, $detectors)) {
             // ascending priority: keys from later detectors will overwrite earlier
-            return (new Detectors\Composite([
+            $all = (new Detectors\Composite([
                 new Detectors\Host(),
                 new Detectors\OperatingSystem(),
                 new Detectors\Process(),
                 new Detectors\ProcessRuntime(),
-                new Detectors\Container(),
                 new Detectors\Sdk(),
                 new Detectors\SdkProvided(),
                 new Detectors\Composer(),
                 new Detectors\Environment(),
```

Comment:
```suggestion
                ...Registry::resourceDetectors(),
                new Detectors\Environment(),
```
Should merge env resource after merging `Registry::resourceDetectors()`.